src/System.Data.SqlClient/src/System.Data.SqlClient.csproj

```diff
@@ -466,6 +466,9 @@
       <LogicalName>System.Data.SqlClient.SqlMetaData.xml</LogicalName>
     </EmbeddedResource>
   </ItemGroup>
+  <ItemGroup>
+    <Compile Include="System\Data\SqlClient\SqlCredential.cs" />
```

Comment:
This compile include should be in the item group ending at line 204 https://github.com/joelbraun/corefx/blob/4273298cf9eaaef061215ab3ffb703541ae81a81/src/System.Data.SqlClient/src/System.Data.SqlClient.csproj#L203


This is also causing a build failure in the CI